web-admin/src/routes/-/github/auth/retry/+page.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+  import { Button } from "@rilldata/web-common/components/button";
+  import { onMount } from "svelte";
+  import { ADMIN_URL } from "../../../../../client/http-client";
+
+  let remote;
+  let githubUsername;
+  let user;
+
+  async function init() {
+    const urlParams = new URLSearchParams(window.location.search);
+    remote = urlParams.get("remote");
+    githubUsername = urlParams.get("githubUsername");
+    const response = await fetch(ADMIN_URL + "/v1/users/current", {
+      method: "GET",
+      credentials: "include",
+    });
```

Comment:
Use the auto-generated client for this – search for `createAdminServiceGetCurrentUser` in other components

Reply:
Applies a few other places